schemas/api/fdc3UserInterfaceHandshake.schema.json

```diff
@@ -1,27 +1,27 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://fdc3.finos.org/schemas/next/api/iframeHandshake.schema.json",
-  "title": "iframe Handshake",
-  "description": "Handshake message sent back to an iframe from the DA proxy code (setup by `getAgent()`) over the `MessagePort` provide in the preceding iFrameHello message, confirming that it is listening to the `MessagePort` for further communication.",
+  "$id": "https://fdc3.finos.org/schemas/next/api/fdc3UserInterfaceHandshake.schema.json",
+  "title": "Fdc3 UserInterface Handshake",
+  "description": "Handshake message sent back to a user interface from the DA proxy code (setup by `getAgent()`) over the `MessagePort` provide in the preceding iFrameHello message, confirming that it is listening to the `MessagePort` for further communication.",
```

Comment:
No big deal, but if you wanted to be consistent with the descriptions in the other schemas you could say "a UI" instead of "a user interface".